employeeApp/src/components/EmployeeForm/EmployeeForm.tsx

```diff
@@ -0,0 +1,325 @@
+                  </div>
+                </div>
+
+                <div className="form-card_group">
+                  <label className="form-card_label">Start Date </label>
+                  <input
+                    type="date"
+                    name="startDate"
+                    className="form-card_control"
+                    value={startDate}
+                    onChange={(e) => setStartDate(e.target.value)}
+                  />
+                </div>
+
+                <div className="form-card_group">
+                  <label className="form-card_label">Finished Date</label>
+                  <input
+                    type="date"
+                    name="finishedDate"
+                    className="form-card_control"
+                    value={finishedDate}
+                    onChange={(e) => setFinishedDate(e.target.value)}
+                    required
+                  />
+                </div>
+
+                <div className="form-card_group">
+                  <label className="form-card_label">
+                    Is this on a full time or part time basis?
+                  </label>
+
+                  <div className="form-card_radio">
+                    <input
+                      type="radio"
+                      id="fullTime"
+                      name="workType"
+                      value="fullTime"
+                      checked={workType === "fullTime"}
+                      onChange={handleWorkTypeChange}
+                    />
+                    <label htmlFor="fullTime">Full-Time</label>
+                  </div>
+
+                  <div className="form-card_radio">
+                    <input
+                      type="radio"
+                      id="partTime"
+                      name="workType"
+                      value="partTime"
+                      checked={workType === "partTime"}
+                      onChange={handleWorkTypeChange}
+                    />
+                    <label htmlFor="partTime">Part-Time</label>
+                  </div>
+                </div>
+
+                <div className="form-card_group">
+                  <label className="form-card_label">Hours per week</label>
+                  <input
+                    type="text"
+                    placeholder="35"
+                    name="workHours"
+                    className="form-card_control"
+                    value={workHours}
+                    onChange={(e) => setWorkHours(e.target.value)}
+                    required
+                  />
+                </div>
+
+                {/* submission buttons */}
+
+                <div className="form_btn">
+                  <button type="submit" className="form_btn_save">
+                    Save
+                  </button>
+                  <Link to="/employees">
+                    <button className="form_btn_cancel">Cancel</button>
+                  </Link>
+                </div>
+              </form>
```

Comment:
render form errors for each field, so you can give feedback to the user

Reply:
am i bit lost - can we please discuss?

Reply:
reimplemented code to use useForm instead of useState. errors now implemented to give feedback to the user through the useForm hook